enterprise/api/v1/views.py

```diff
@@ -318,12 +319,13 @@ def enroll_learners_in_courses(self, request, pk):
                 ))
                 track_enrollment(PATHWAY_CUSTOMER_ADMIN_ENROLLMENT, request.user.id, course_run)
                 if serializer.validated_data.get('notify'):
-                    enterprise_customer.notify_enrolled_learners(
-                        catalog_api_user=request.user,
-                        course_id=course_run,
-                        users=pending_users | existing_users,
-                        admin_enrollment=True,
+                    email_items = enterprise_customer.prepare_notification_content(
```

Comment:
Nit: can you change the name of 'prepare_notification_content' to 'serialize_notification_content' ?